gammapy/irf/background.py

```diff
@@ -135,6 +135,32 @@ def to_2d(self):
     def peek(self, figsize=(10, 8)):
         return self.to_2d().peek(figsize)
 
+    def plot_at_energy(self, energy=None, ax=None, add_cbar=True, **kwargs):
+        """ Plot the background rate in Field of view co-ordinates at a given energy.
+
+        Parameters
+        -----------
+        energy : `~astropy.units.Quantity`
```

Comment:
Could it be possible to pass a list of energies and have the result plotted as plot_grid?

Reply:
I can do this, but these are not maps, so have to re-implement a part of the code in `Maps.plot_grid` to make an automatic selection of the number of rows and columns in the grid.